src/lib/QtCore/QObject.ts

```diff
@@ -8,7 +8,8 @@ export abstract class NodeObject extends EventWidget {
         return this.native.inherits(className);
     }
     setProperty(name: string, value: any): boolean {
```

Comment:
change any to a union of all supported values
(all values that Qvariant supports)